sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v1/schema.yaml

```diff
@@ -152,6 +152,9 @@ fields:
   - mode: NULLABLE
     name: dltoken
     type: STRING
+  - name: dlsource
```

Comment:
Admittedly I don't have lots of context here, but what's this one for?

Reply:
Ahh seems like this was just missed in a previous schema update https://github.com/mozilla/bigquery-etl/pull/4240